tests/callbacks/test_curriculum_learning_callback.py

```diff
@@ -1,14 +1,28 @@
 # Copyright 2024 MosaicML LLM Foundry authors
 # SPDX-License-Identifier: Apache-2.0
 
+from omegaconf import OmegaConf as om
+
 from llmfoundry.utils.builders import build_callback
 
 
 def test_curriculum_learning_callback_builds():
```

Comment:
given that we are making it no longer experimental, its complete functionality should be unit tested

Reply:
Done